web/history.jsp

```diff
@@ -279,8 +279,10 @@ revision2 = revision2 >= hist.getHistoryEntries().size() ? hist.getHistoryEntrie
                 %><%= author %><%
                 }
                 %></td>
-            <td><a name="<%= rev %>"></a><p><%
+            <td><a name="<%= rev %>"></a><%
+                int summaryLength = 200;
```

Comment:
this should be probably a configuration variable, care to add it?
so people can at least set it using readonly xml config passed to indexer ?